TestREADME.md

```diff
@@ -0,0 +1,210 @@
+   - Use stub for submissions to DES
+   
+### GET /test-only/create-journey
+#### Deprecated - use GET /test-only/create-sole-trader-journey
+
+---
+This is a test entry point which simulates a service by triggering the initial POST call to set up a journey for a Sole Trader.
+
+1. Continue URL (Required)
+
+    - Where to redirect the user after the journey has been completed
+
+2. Service Name (Optional)
+
+    - Service Name to use throughout the service
+    - Currently, this is empty by default, so the default service name will be used
+
+3. DeskPro Service ID (Required)
+
+    - Used for the `Get help with this page` link
+    - This is currently autofilled but can be changed
+
+4. Sign Out Link (Required)
+
+    - Shown in the HMRC header - typically a link to a feedback questionnaire
+    - This is currently autofilled but can be changed
+
+5. Enable SAUTR Check (Optional)
+
+    - Shows the user an extra page where they can enter an SAUTR
+    - This is currently defaulted to false unless otherwise specified
+    - If this is enabled, refer to Using the Authenticator Stub section below
+
+### GET /test-only/create-sole-trader-journey
+
+---
+This is a test entry point which simulates a service by triggering the initial POST call to set up a journey for a Sole Trader.
+
+1. Continue URL (Required)
+
+   - Where to redirect the user after the journey has been completed
+
+2. Service Name (Optional)
+
+   - Service Name to use throughout the service
+   - Currently, this is empty by default, so the default service name will be used
+
+3. DeskPro Service ID (Required)
+
+   - Used for the `Get help with this page` link
+   - This is currently autofilled but can be changed
+
+4. Sign Out Link (Required)
+
+   - Shown in the HMRC header - typically a link to a feedback questionnaire
+   - This is currently autofilled but can be changed
+
+5. Enable SAUTR Check (Optional)
+
+   - Shows the user an extra page where they can enter an SAUTR
+   - This is currently defaulted to false unless otherwise specified
+   - If this is enabled, refer to Using the Authenticator Stub section below
+
+### GET /test-only/create-individual-journey
+
+---
+This is a test entry point which simulates a service by triggering the initial POST call to set up a journey for an Individual.
+
+1. Continue URL (Required)
+
+   - Where to redirect the user after the journey has been completed
+
+2. Service Name (Optional)
+
+   - Service Name to use throughout the service
+   - Currently, this is empty by default, so the default service name will be used
+
+3. DeskPro Service ID (Required)
+
+   - Used for the `Get help with this page` link
+   - This is currently autofilled but can be changed
+
+4. Sign Out Link (Required)
+
+   - Shown in the HMRC header - typically a link to a feedback questionnaire
+   - This is currently autofilled but can be changed
+   
+
+### GET test-only/retrieve-journey/:journeyId or test-only/retrieve-journey
```

Comment:
the second endpoint takes a query parameter so might be useful explaining what the query parameter is